### PR TITLE
Build changes to docs

### PR DESCRIPTION
**Pull request by MCGallaspy**
The PR that pairs so nicely with https://github.com/learningequality/ka-lite/pull/4075, like a fine wine and stinky cheese.

### CODE REVIEW

**Comment by jamalex** [comment]
Quick question on this: what's the plan/timeline for migrating this to a cron-based build process? (rather than being built directly through Ansible commands -- which are meant to be used for system configuration, not for initiating long-running processes directly). No immediate rush on that, but just wanted to make sure it was still on the radar.

**Comment by MCGallaspy** [comment]
Yeah, I haven't forgotten about it. It's a moderately complicated process at this point, so it would probably take a day or two to port over & debug -- meaning making a script template and making sure it works without issues. The only question is when you want me to do it: I can do it now and push back my other planned activities, or I can add it to my future to-do list.

**Comment by jamalex** [comment]
I'd say add it to the future to-do list. As mentioned, I just wanted to make sure it was still on the radar (before too much work went into further fleshing out the current approach).

File: field.yml [hunk 1]
```diff
@@ -2,13 +2,31 @@
...
+    - kalite_docs_dir: docs
+    - kalite_docs_reqs: requirements_sphinx.txt
+    - extra_sphinx_opts:
   roles:
     - common
     - server
     - docsbuilder
-    - { role: docsbuilder, ka_lite_distributed_git_branch: develop, docs_html_host_dir: /usr/share/nginx/html/develop/docs, docs_pdf_host_dir: /usr/share/nginx/html/develop/docspdf }
\ No newline at end of file
+    - { role: docsbuilder,
```
Comment (jamalex) [inline field.yml]: Note (but not a blocker): YAML doesn't need braces

**Pull request merged**